src/plugins/intel_gpu/tests/fusions/fully_connected_fusion_test.cpp

```diff
@@ -81,14 +82,23 @@ class FullyConnectedFusingTestOneDNN : public BaseFusingTest<fully_connected_tes
         auto input_prim = p.data_type == data_types::u8 ? get_mem(get_input_layout(p), 0, 10) : get_mem(get_input_layout(p));
 
         auto impl_forcing = cfg_fused.get_property(ov::intel_gpu::force_implementations);
-
         auto forcing_format = p.input_format;
         for (auto& forcing : impl_forcing)
             if (forcing.first == "fc_prim")
                 forcing_format = forcing.second.output_format;
 
-        ov::intel_gpu::ImplementationDesc conv_impl = { forcing_format, "", impl_types::onednn };
-        cfg_fused.set_property(ov::intel_gpu::force_implementations(ov::intel_gpu::ImplForcingMap{ { "fc_prim", conv_impl } }));
+        ov::intel_gpu::ImplementationDesc fc_impl = { forcing_format, "", impl_types::onednn };
+        cfg_fused.set_property(ov::intel_gpu::force_implementations(ov::intel_gpu::ImplForcingMap{ { "fc_prim", fc_impl } }));
+
+        if (!p.ocl_kernel_name.empty()) {
```

Comment:
is `ocl_kernel_name` required? what about just forcing ref kernel for unfused case?

Reply:
If use force_implemention keyword, network always run with optimization path(prepare_qunatization, primitive_fusing, reorder_input...).
The purpose of fusing test is comparing non_fused vs fused network, but we can't guarantee pure none fused network if use force_implementaton in non_fused network.
FC and eltwise fusing in fp16 is currently blocked in clDNN.  So this cases is okay whether non_fused network use force_impl or not, but other tests used FullyconnectedFusingTestOneDNN have the issue for ref value correctness.
* Solution for removing ocl_kernel_name
1. Find and fix the high priority kernel(fully_connected_gpu_bs_f_bsv16_b1) failed reason and then run the high priority kernel without ocl_kernel_name force_implementation.
2. Make another FullyConnectedFusingTestOneDNN class only for FC and eltwise fusion. It always select ref kernel.
3. Nothing action